vagrant/forum/forum.py

```diff
@@ -40,68 +38,26 @@
 
 # HTML template for an individual comment
 POST = '''\
-    <div class=post><em class=date>%(time)s</em><br>%(content)s</div>
+    <div class=post><em class=date>%s</em><br>%s</div>
```

Comment:
Are we sticking with %s for string interpolation over using `.format()`?   f-strings are now part of Python 3.6 (https://www.python.org/dev/peps/pep-0498/), so I wonder if using the `.format()` will help future proof the code a little. 

I think the %s interpolation is in the course content, though, so there's an argument for keeping it as is for now.